utils/scapp.h

```diff
@@ -554,18 +655,11 @@ scApp::processOption(argparser& parser, int opt)
         break;
       case 't':
         options.bopts.threadCount = strtoi(parser.optarg.c_str());
+		options.astcopts.threadCount = strtoi(parser.optarg.c_str());
```

Comment:
Too many spaces.

Reply:
There aren't there in last commit. If you just look at the file itself (make sure there is no "diff+commit-sha" in the URL. This confused me for a bit.